dist/ads.js

```diff
@@ -127,14 +132,26 @@ const parseSellersJSON = async (response) => {
             seller_count: 0,
           }
         },
-        passthrough_count: 0
+        passthrough_count: 0,
+        confidential_count: 0
       }
     };
 
     // Clean up file content
     result.seller_count = content.sellers.length;
 
     for (let seller of content.sellers) {
+      // Validating records
+      if (!seller.seller_type && !seller.domain) {
```

Comment:
```suggestion
      if (!seller.seller_type || !seller.id) {
```
seller.domain is not a required field, but seller.id is.